pmoves/services/yt-cookie-writer/main.py

```
@@ -0,0 +1,168 @@
+async def fetch_and_write_cookies(user_id: str = DEFAULT_USER_ID) -> bool:
+    """Fetch encrypted cookies from Supabase, decrypt, write to disk.
+
+    Returns True on success.
+    """
+    import httpx
+
+    url = f"{_supabase_url()}{TABLE_PATH}?user_id=eq.{user_id}&select=encrypted_cookies"
+    try:
+        resp = httpx.get(url, headers=_supabase_headers(), timeout=10)
+        rows = resp.json() if resp.status_code == 200 else []
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)